src/core/respec.js

```diff
@@ -0,0 +1,24 @@
+// @ts-check
+/**
+ * This module adds a `respec` object to the `document` with the following
+ * readonly properties:
+ *  - version: returns version of ReSpec Script.
+ *  - ready: an alias to `document.respecIsReady`.
+ */
+
+export const name = "core/respec";
```

Comment:
Should we call it something else? Like "core/respec-global"?

Reply:
Or maybe even `core/document.respec`

Reply:
"core/respec-global" sounds good